### PR TITLE
Feature/13 - Fixes scrollbar misalignment 

### PR DESCRIPTION
**Pull request by jrmartin**
Removes scrollbar from canvas element, not needed to scroll canvas.

### CODE REVIEW

File: src/test-app/app.js [hunk 1]
```diff
@@ -193,7 +203,7 @@ const fileExtensionRe = /(?:\.([^.]+))?$/;
         }
 
         footer{
-            margin-top: 10px;
+            margin-top: 1vh;
```
Comment (zsinnema) [inline src/test-app/app.js:206]: why not stick the footer to the bottom like:
```
position: absolute;
bottom: 0;
width: 100%;
```

**Review by zsinnema — CHANGES_REQUESTED** [review]

**Review by zsinnema — APPROVED** [review]
lgtm

**Comment by tgbugs** [comment]
@jrmartin I think this removed the scroll bars that are in the resources tab so it is no longer possible to scroll through the tables in that tab.

**Comment by jrmartin** [comment]
@tgbugs I reopened the PR with a fix for it #27. This branch is not based on the open PR with all the fixes you did to merge the different PR's yet (branch : tgbugs/big-merge), we haven't sync that one yet #26.

I can open a PR to tgbugs/big-merge with the fix too if you prefer.

**Pull request merged**